src/m365/spo/commands/serviceprincipal/serviceprincipal-permissionrequest-list.ts

```diff
@@ -58,6 +71,40 @@ class SpoServicePrincipalPermissionRequestListCommand extends SpoCommand {
       this.handleRejectedPromise(err);
     }
   }
+
+  private async getOAuth2PermissionGrants(spoClientExtensibilityWebApplicationPrincipalId: string): Promise<string[] | null> {
+    const requestOptions: CliRequestOptions = {
+      url: `https://graph.microsoft.com/v1.0/oAuth2Permissiongrants/?$filter=clientId eq '${spoClientExtensibilityWebApplicationPrincipalId}' and consentType eq 'AllPrincipals'`,
+      headers: {
+        accept: 'application/json;odata.metadata=none'
+      },
+      responseType: 'json'
+    };
+
+    const response: { value: OAuth2PermissionGrant[] } = await request.get<{ value: OAuth2PermissionGrant[] }>(requestOptions);
```

Comment:
Here we could use the ODataResponse type, like `ODataResponse<OAuth2PermissionGrant>`